CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## [1.0.0-rc.11] - 2023-09-07
+
+We've received some valuable feedback and improved some aspects of SDK API in
+this release. Most of the changes are just extensions and only a few are
+incompatible with the previous RC (except for scheduled deprecated Pool API
+removal). Try it out and leave feedback. We don't currently have any API
+changes planned, so this might become 1.0 as is (not counting documentation
+improvements that will be done).
+
+New features:
+ * SignedData method for signed structures (#513)
+ * CopyTo method for deep structure copies (#512)
+
+Behaviour changes:
+ * ObjectRangeInit and ObjectRangeInit unification between Client and Pool
+   (#491)
+ * removal of all deprecated Pool APIs (#491)
+ * ObjectHead method now returns header instead of intermediate struct (#497)
+ * ObjectListReader now returns error from Read instead of bool flag
+ * Object's Sign/VerifySignature methods replacing
+   CalculateAndSetSignature/VerifyIDSignature, harmonizing this API with other
+   structures (#513)
+ * System role is ignored in the EACL matcher now (#515)
```

Comment:
```suggestion
 * System role is ignored in the eACL matcher now (#515)
```
seems like we prefer this spelling usually. not critical

Reply:
OK, fixed.